cppcache/src/Log.cpp

```diff
@@ -60,6 +61,8 @@ static FILE* g_log = nullptr;
 
 static std::string g_hostName;
 
+static thread_local std::string g_threadName;
```

Comment:
See https://google.github.io/styleguide/cppguide.html#thread_local and https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables for naming guidelines. I think you want `kThreadName` but I am not too keen on that given it isn't really a constant.

Reply:
Leaving this one to be consistent with name of several other variables in this file.